package-system/AWSGameLiftServerSDK/install-linux.sh

```diff
@@ -0,0 +1,67 @@
+#!/bin/bash
+#
+# Copyright (c) Contributors to the Open 3D Engine Project.
+# For complete copyright and license terms please see the LICENSE at the root of this distribution.
+#
+# SPDX-License-Identifier: Apache-2.0 OR MIT
+#
+
+echo "TEMP_FOLDER=${TEMP_FOLDER}"
+echo "TARGET_INSTALL_ROOT=${TARGET_INSTALL_ROOT}"
+
+# Get the path within the source tarball to the root of the source folder for the build
+SDK_SRC_SUBPATH=${1:-.}
+
+# Get the install path from the build as the source for the final install package
+SRC_INSTALL_PACKAGE_PATH=${TEMP_FOLDER}/install/
+if [ ! -d ${SRC_INSTALL_PACKAGE_PATH} ]
+then
+	echo "Invalid source package path ${SRC_INSTALL_PACKAGE_PATH}"
+	exit 1
+fi
+
+# Copy an additional notice file
+ADDITIONAL_NOTICE_FILE=${TEMP_FOLDER}/src/${SDK_SRC_SUBPATH}/NOTICE_C++_AMAZON_GAMELIFT_SDK.TXT
+if [ ! -f ${ADDITIONAL_NOTICE_FILE} ]
+then
+	echo "Invalid source package path ${SRC_INSTALL_PACKAGE_PATH}"
+	exit 1
+fi
+echo "Copying the additional notice file ${ADDITIONAL_NOTICE_FILE} -> ${TARGET_INSTALL_ROOT}"
+cp ${TEMP_FOLDER}/src/${SDK_SRC_SUBPATH}/NOTICE_C++_AMAZON_GAMELIFT_SDK.TXT ${TARGET_INSTALL_ROOT}
+
+
+# Copy folder with messaging and error checks
+# 
+# Arguments:
+#   $1 : Source folder
+#   $2 : Destination folder
+copy_folder() {
+
+    SRC_FOLDER=$1
+    TGT_FOLDER=$2
+    if [ ! -d ${SRC_FOLDER} ]
+    then
+	    echo "Invalid source folder copy path ${SRC_FOLDER}"
+	    exit 1
```

Comment:
nit: some tabs may have snuck into this file

Reply:
Definitely, there are some tabs in this file.
Our tab validator only runs on the main o3de repo